frontend/website/src/HoverCard.re

```diff
@@ -0,0 +1,101 @@
+module Styles = {
+  open Css;
+
+  let ctaButton =
+    style([
+      background(Style.Colors.hyperlinkAlpha(0.3 *. 0.2)),
+      borderRadius(`px(6)),
+      textDecoration(`none),
+      padding(`rem(1.43)),
+      paddingTop(`rem(1.625)),
+      margin(`auto),
+      height(`rem(14.25)),
+      width(`rem(21.25)),
+      media(
+        "(min-width: 60rem)",
+        [height(`rem(12.5)), width(`rem(23.8))],
+      ),
+      media(
+        "(min-width: 105rem)",
+        [height(`rem(11.5)), width(`rem(32.5))],
+      ),
+      hover([
+        backgroundColor(`hex("FAFCFD")),
+        cursor(`pointer),
+        boxShadow(
+          ~x=`zero,
+          ~y=`px(4),
+          ~blur=`px(8),
+          ~spread=`zero,
+          `rgba((0, 0, 0, 0.25)),
+        ),
+        selector("svg", [SVG.fill(Style.Colors.hyperlink)]),
+      ]),
+    ]);
+
+  let ctaContent = style([display(`flex)]);
+
+  let ctaText = style([width(`percent(100.))]);
+
+  let ctaHeading =
+    merge([
+      Style.H2.basic,
+      style([
+        fontWeight(`bold),
+        color(Style.Colors.marine),
+        textAlign(`left),
+        paddingBottom(`rem(0.3)),
+      ]),
+    ]);
+
+  let ctaBody =
+    merge([
+      Style.Body.basic,
+      style([marginTop(`rem(1.)), color(Style.Colors.midnight)]),
+    ]);
+
+  let ctaIcon =
+    style([
+      marginTop(`px(2)),
+      minWidth(`px(36)),
+      maxHeight(`px(48)),
```

Comment:
if you use pixels here does this still look okay when you zoom in and out?

Reply:
yes it's all ok